editor/editor_node.cpp

```diff
@@ -3702,6 +3702,18 @@ void EditorNode::fix_dependencies(const String &p_for_file) {
 
 int EditorNode::new_scene() {
 	int idx = editor_data.add_edited_scene(-1);
+	// Remove [empty] scene if any other scene is loaded.
+	if (editor_data.get_edited_scene_count() > 1) {
+		for (int i = 0; i < editor_data.get_edited_scene_count() - 1; i++) {
+			bool unsaved = get_undo_redo()->is_history_unsaved(editor_data.get_scene_history_id(i));
+			if (!unsaved && ((*editor_data.get_scene_path(i).get_data()) == 0)) {
```

Comment:
```suggestion
			if (!unsaved && editor_data.get_scene_path(i).get_data().is_empty()) {
```
This is how you should check for empty string.

Reply:
But ,
editor_data.get_scene_path(i).get_data() is of type = const char32_t * ...

Reply:
Sorry, typo.

Reply:
This is still unresolved, I edited my suggestion.

Reply:
sorry ,
i haven't noticed that .

Reply:
Changes Done .